server/fleet/software.go

```diff
@@ -193,6 +193,22 @@ type SoftwareTitleListOptions struct {
 	TeamID              *uint `query:"team_id,optional"`
 	VulnerableOnly      bool  `query:"vulnerable,optional"`
 	AvailableForInstall bool  `query:"available_for_install,optional"`
+	SelfServiceOnly     bool  `query:"self_service,optional"`
+}
+
+type HostSoftwareTitleListOptions struct {
+	// ListOptions cannot be embedded in order to unmarshal with validation.
```

Comment:
thanks for the comment, I was wondering why

Reply:
That's something I took from another list options struct, I think it's newish following some endpoint-ops changes that added more strict query param validations.

Reply:
thanks, I will keep that in mind for future endpoints.